addons/pos_epson_printer/views/pos_printer_views.xml

```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
-    <record id="view_restaurant_printer_iot_form" model="ir.ui.view">
-        <field name="name">pos.restaurant.iot.config.form.view</field>
-        <field name="model">restaurant.printer</field>
-        <field name="inherit_id" ref="pos_restaurant.view_restaurant_printer_form"/>
+    <record id="view_pos_printer_iot_form" model="ir.ui.view">
```

Comment:
The module doesn't depends of iot. 
This view shouldn' include iot in its name.